lib/ecom/api.ts

```diff
@@ -37,28 +32,7 @@ function getWixClientId() {
     return env.WIX_CLIENT_ID ?? DEMO_STORE_WIX_CLIENT_ID;
 }
 
-function ensureSessionIntegrity() {
-    const sessionWixClientId = Cookies.get(WIX_CLIENT_ID_COOKIE_KEY);
-    const configuredWixClientId = getWixClientId();
-
-    // Clear user session if headless site changed.
-    // This will clear old cart if it exists.
-    // We have to do this because old cart may contain products from old site.
-    if (sessionWixClientId !== configuredWixClientId) {
-        Cookies.remove(WIX_SESSION_TOKEN_COOKIE_KEY);
-    }
-
-    Cookies.set(WIX_CLIENT_ID_COOKIE_KEY, configuredWixClientId);
-}
-
-function getTokensClient() {
-    ensureSessionIntegrity();
-
-    const tokens = Cookies.get(WIX_SESSION_TOKEN_COOKIE_KEY);
-    return tokens ? JSON.parse(tokens) : undefined;
-}
-
-function getWixClient() {
+function createApiWixClient(tokens?: Tokens) {
```

Comment:
`createWixSDKClient` or `createWixClient` to avoid having `createApiWixClient` and `createApi`?